### ISSUE TITLE
Number parsing exceptions

### ISSUE THREAD

**Issue by hboylan** [spencermountain/compromise#461]
So far I've had pretty good results using this library to parse numbers. Unfortunately, I've started to see some odd edge cases with verbose pronunciations like:

```javascript
const nlp = require('compromise')

// works
nlp('fifty five').values().toNumber().out() // 55

// doesn't work
nlp('forty eight').values().toNumber().out() // 40 8
```

Preventing me from using this in production... :sob:

**Comment by TSavo** [comment]
I wrote this for this very reason: https://www.npmjs.com/package/english2number

**Comment by hboylan** [comment]
Solid suggestion. This issue appears to only occur in the 40s range... Trying to figure out how to add a custom lexicon to patch this locally for now.

**Comment by spencermountain** [comment]
hey Hugh! whoa nice bug. Surprised that one got through.
I'll take a look.

hey @TSavo nice work!

**Comment by spencermountain** [comment]
hahaha
![image](https://user-images.githubusercontent.com/399657/37182300-97d32690-22fe-11e8-8089-cd36fd28a04e.png)
oops!

**Comment by hboylan** [comment]
Ha, awesome! Wondered if that might be the case since I made the same typo a couple times.

Really appreciate you looking at this so quickly Spencer!

**Comment by spencermountain** [comment]
fixed in 11.5.3, lemme know if you see something else.
cheers

**Comment by hboylan** [comment]
Confirmed! You're the man! :100: 
```javascript
nlp('forty two').values().toNumber().out() // 42
```
